### PR TITLE
Update VPN guarantee copy (Fixes #10696)

### LINKED ISSUE #10696: VPN landing page and/or FAQs should call out that the 30 day money back guarantee is only for purchases on the website
## Description

Our landing page calls out the 30 day money back guarantee prominently. Our FAQs call out the 30 day money back guarantee, but does not provide info that purchases from app stores are not included.

Details and suggestions for how to fix are available at this Jira ticket: https://mozilla-hub.atlassian.net/browse/VPN-1173

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Updates VPN 30-day money back guarantee copy, across all VPN web pages that mention it.
- Updates FAQ copy to highlight that this is only available through subscriptions on the website.

## Issue / Bugzilla link
#10696

## Testing
- http://localhost:8000/en-US/products/vpn/
- http://localhost:8000/en-US/products/vpn/more/what-is-a-vpn/
- http://localhost:8000/en-US/products/vpn/more/what-is-an-ip-address/
- http://localhost:8000/en-US/products/vpn/more/when-to-use-a-vpn/
- http://localhost:8000/en-US/products/vpn/more/vpn-or-proxy/
- http://localhost:8000/en-US/products/vpn/mobile/
- http://localhost:8000/en-US/products/vpn/desktop/
- http://localhost:8000/en-US/products/vpn/desktop/mac/
- http://localhost:8000/en-US/products/vpn/desktop/windows/
- http://localhost:8000/en-US/products/vpn/desktop/linux/
- http://localhost:8000/en-US/products/vpn/mobile/ios/
- http://localhost:8000/en-US/products/vpn/mobile/android/

### CODE REVIEW

File: l10n/en/products/vpn/landing.ftl [hunk 1]
```diff
@@ -159,7 +159,12 @@ vpn-landing-faq-compatibility-question-desc-linux-v2 = <a href="{ $url }">{ -bra
 vpn-landing-faq-compatibility-question-desc-linux = { -brand-name-linux } ({ -brand-name-ubuntu }-only)
 
 vpn-landing-faq-refund-question-heading = What is { -brand-name-mozilla-vpn }’s refund policy?
+
+vpn-landing-faq-refund-question-desc-v2 = You can get your money back within 30 days of purchasing your subscription through your Firefox Account or through Firefox, from this website. Contact us and submit the refund request by tapping the “Get Help” button in Settings on your { -brand-name-mozilla-vpn } app.
```
Comment (alexgibson) [inline l10n/en/products/vpn/landing.ftl:163]: Note to self: these need to be brand name variables

**Comment by alexgibson** [comment]
This is ready for review.

File: bedrock/products/templates/products/vpn/includes/pricing-variable.html [hunk 2]
```diff
@@ -17,7 +17,7 @@ <h3 class="vpn-pricing-variable-sub-heading">{{ ftl('vpn-shared-pricing-variable
     <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-encrypt') }}</li>
     <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-bandwidth') }}</li>
     <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-activity') }}</li>
-    <li class="vpn-feature-list-item">{{ ftl('vpn-shared-features-guarantee', fallback='vpn-shared-money-back-guarantee') }}</li>
+    <li class="vpn-feature-list-item vpn-features-guarentee">{{ ftl('vpn-shared-features-guarantee', fallback='vpn-shared-money-back-guarantee') }}</li>
```
Comment (maureenlholland) [inline bedrock/products/templates/products/vpn/includes/pricing-variable.html:20]:
```suggestion
    <li class="vpn-feature-list-item vpn-features-guarantee">{{ ftl('vpn-shared-features-guarantee', fallback='vpn-shared-money-back-guarantee') }}</li>
```

File: media/css/products/vpn/common.scss [hunk 3]
```diff
@@ -35,6 +35,34 @@ $image-path: '/media/protocol/img';
...
+
+        &:hover,
+        &:active,
+        &:focus {
+            text-decoration: underline;
+        }
+    }
+}
+
+.vpn-features-guarentee {
```
Comment (maureenlholland) [inline media/css/products/vpn/common.scss:59]:
```suggestion
.vpn-features-guarantee {
```

**Review by maureenlholland — APPROVED** [review]
css naming typo nits, not blocking

r+ ✅

**Comment by alexgibson** [comment]
@maureenlholland thanks for the review, updated

**Review by maureenlholland — APPROVED** [review]
r++ nice work!

**Pull request merged**